src/components/map/EventLayer.js

```diff
@@ -119,11 +131,13 @@ class EventLayer extends Layer {
         ) : null;
     }
 
-    onEventClick({ feature, coordinates }) {
-        this.setState({ popup: { feature, coordinates } });
+    onEventClick({ feature, coordinates, offset }) {
+        this.setState({ popup: { feature, coordinates, offset } });
```

Comment:
Question: where does this offset come from?

Reply:
It comes from Maps GL: https://github.com/dhis2/maps-gl/blob/feat/event-layer/src/layers/Spider.js#L82

It is the offset from the spiderified marker to the clicked feature which don't have a coordinate.

Reply:
OK, that's fine I just wasn't sure where it came from

Reply:
Smaller API footprint is better, so I've now removed the need for the offset parameter in Maps GL. The solution was to add the offset to the [lng, lat] position directly and return it with the click event.